src/modules/cmdpal/ext/Microsoft.CmdPal.Ext.Apps/AllAppsSettings.cs

```diff
@@ -19,6 +19,16 @@ public class AllAppsSettings : JsonSettingsManager
 
     private static string Experimental(string propertyName) => $"{_namespace}.experimental.{propertyName}";
 
+    private static readonly List<ChoiceSetSetting.Choice> _searchResultLimitChoices =
+    [
+        new ChoiceSetSetting.Choice(Resources.limit_none, Resources.limit_none),
```

Comment:
I don't think it's a good idea to persist localized strings as a setting value:

At least

```suggestion
        new ChoiceSetSetting.Choice(Resources.limit_none, "-1"),
```

or create an enum for this.

Reply:
Agreed